python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,44 @@
+from ray._private.structured_logging.formatters import LogfmtFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+
+# A dictionary to map encoding types to their corresponding logging configurations.
+LOG_MODE_DICT = {
+    "TEXT": {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "logfmt": {
+                "()": LogfmtFormatter,
+            },
+        },
+        "filters": {
+            "core_context": {
+                "()": CoreContextFilter,
```

Comment:
"The key '()' has been used as the special key because it is not a valid keyword parameter name, and so will not clash with the names of the keyword arguments used in the call. The '()' also serves as a mnemonic that the corresponding value is a callable."

Ref: https://docs.python.org/3/library/logging.config.html#logging-config-dictschema